src/Esfa.Vacancy.Register.Infrastructure/VacancyRegisterRetryPolicy.cs

```diff
@@ -0,0 +1,16 @@
+﻿using System;
+using Polly;
+
+namespace Esfa.Vacancy.Register.Infrastructure
+{
+    public static class VacancyRegisterRetryPolicy
+    {
+        public static Policy GetFixedIntervalPolicy(Action<Exception, TimeSpan, int, Context> onRetry)
+        {
+            return Policy
+                .Handle<Exception>()
+                .WaitAndRetryAsync(3, (attempt) => TimeSpan.FromMilliseconds(500),
```

Comment:
Could you plonk the 3 and the 500ms things in constants.

Reply:
Is it worth making wait time configurable? Perhaps allowing it to be different in test environments?